src/js/integrations/debugsymbolicator.ts

```diff
@@ -28,6 +30,11 @@ export type ReactNativeError = Error & {
   componentStack?: string;
 };
 
+type ErrorLike = {
+  stack: string
+};
```

Comment:
Let's move this type to `utils/error`

Reply:
I removed it since it is no longer required with the new changes.